config.json

```diff
@@ -2,7 +2,10 @@
     "name": "User Profile",
     "namespace": "UserProfile\\ExternalModule",
     "description": "Provides user profiles management - in other words, it gives the administrators the ability to create (and manage) additional user account information such as address, country of birth, job position, etc.",
-    "permissions": [],
+    "permissions": [
+        "hook_every_page_before_render",
```

Comment:
If there was a way to make comments on json files I would say that this permission is necessary to make UserProfile class available at anytime. Otherwise the following code won't work for all cases:
```
use UserProfile\UserProfile;
```